pytorch_lightning/strategies/ddp.py

```diff
@@ -196,6 +196,18 @@ def pre_configure_ddp(self):
             )
             self._ddp_kwargs["find_unused_parameters"] = True
 
+        if self.root_device.type == "hpu":
+            self._static_graph = False
```

Comment:
declare `sel._static_graph` in the constructor for typechecking purposes.

Reply:
updated to hpu plugin